notes/v1.0.0-beta2.md

```diff
@@ -0,0 +1,142 @@
+# Changelog
```

Comment:
Would it maybe make sense to remove this Markdown file or replace the content with a pointer to the official docs? Ideally, we'd have this information in just one place and get people to look in the official docs. 

(If not, it would be good to include the suggested edits I made for the Asciidoctor version.)

Reply:
It might be a good idea to redirect people to the release notes directly with an anchor to the version specific contents. I think we should do that from now on.